Force LC_ALL=C before running df -l command on pki-enroll-request

nkovacne opened the pull request:
## Changes introduced with this PR

* pki-enroll-request.sh checks if the CA file is stored on a local filesystem. However, the 'df' command output depends on the local system language, thus output may vary. With this patch LANG=C is forced just before the 'df' command is run.

## Are you the owner of the code you are sending in, or do you have permission of the owner?

Yes.

nkovacne commented on packaging/bin/pki-enroll-request.sh:
```diff
@@ -129,7 +129,7 @@ done
 
 LOCK="${PKIDIR}/${CA_FILE}".pem
-df -l "${LOCK}" 2> /dev/null | grep -q "File" || die "${LOCK} is not on a local filesystem"
+LANG=C df -l "${LOCK}" 2> /dev/null | grep -q "File" || die "${LOCK} is not on a local filesystem"
```
I tried it and it indeed works, however reading the documentation it seems more reasonable to use LC_ALL, I will change it.

didib replied:
I meant whether you tried it with LC_MESSAGES set, as opposed to only with LANG being set. Sorry I wasn't clear.

nkovacne replied:
With LC_MESSAGES it won't work, as LANG only works if no LC_* variables are set, it doesn't override anything. I also tried it with LC_ALL=C and it works.

didib commented:
LGTM. Please handle the DCO failure - add a 'Signed-off-by:' line to your commits. And perhaps squash to a single commit.

nkovacne commented:
> LGTM. Please handle the DCO failure - add a 'Signed-off-by:' line to your commits. And perhaps squash to a single commit.

Removed the useless commit, and added the Signed-off-by line to the rebased commit.

didib commented:
Sorry, github insists on using the same "real name" in both the "Author" (and perhaps Commit) fields and in the "Signed-off-by" line. For former you have "Nicolás Kovac Neumann" but for latter you dropped "Neumann". See:

https://github.com/oVirt/ovirt-engine/pull/891/checks?check_run_id=18697876068

Please fix, thanks...

didib commented:
Thanks!
